CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# [0.3.1](https://github.com/VeryGoodOpenSource/pub_updater/compare/v0.3.0...v0.3.1) (2023-06-19)
+
 # [0.3.0](https://github.com/VeryGoodOpenSource/pub_updater/compare/v0.2.4...v0.3.0) (2023-04-11)
```

Comment:
Am I missing something?

Reply:
Chores and refacttors are no included by cenventional-changelog. Once we moved to the release workflows this whole file chouls change IMO.


We have precedent on releases as this: https://github.com/VeryGoodOpenSource/pub_updater/blob/main/CHANGELOG.md#011-2021-09-13